workspaces/mujoco_workspace.py

```diff
@@ -83,11 +83,8 @@ def train(self):
             if done:
                 values = np.array(val_seq)
                 ret = np.array(rew)
-                for i in range(len(rew)):
-                    s = 0
-                    for j in range(len(rew) - i):
-                        s += rew[j]*self.agent.gamma**j
-                    ret[i] = s
+                for i in range(len(rew)-2, -1, -1):
+                    ret[i] +=rew[i] +ret[i+1] * self.agent.gamma
```

Comment:
Now you are adding rew[i] twice, it is already in ret[i] becuase you created the array from reward list